app/Makefile

```diff
@@ -54,4 +54,9 @@ storybook: # Run the Storybook local dev server in Docker
 	docker compose logs --follow storybook
 
 stop:
-	docker compose down
\ No newline at end of file
+	docker compose down
+
+
+lint-github-actions: # Run actionlint on the .github folder
+	@echo "Running actionlint on the .github folder..."
+	@actionlint -color
```

Comment:
Should we include this? I kept running into shellcheck issues on the PR - would be nice to just run locally.